Sources/FoundationEssentials/String/String+Internals.swift

```diff
@@ -35,15 +40,42 @@ extension String {
         // leading slash indicates a rooted path on the drive for the current
         // working directory.
         return try Substring(self.utf8.dropFirst(bLeadingSlash ? 1 : 0)).withCString(encodedAs: UTF16.self) { pwszPath in
+            if relative && PathIsRelativeW(pwszPath) {
+                return try body(pwszPath)
+            }
+
             // 1. Normalize the path first.
+            // Contrary to the documentation, this works on long paths independently
+            // of the registry or process setting to enable long paths (but it will also
```

Comment:
Ah, it states the truth on [this page](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats) instead of the docs for GetFullPathName itself:

> You can pass paths of more than MAX_PATH characters to [GetFullPathName](https://learn.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-getfullpathnamea) without \\?\. It supports arbitrary length paths up to the maximum string size that Windows can handle.